packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -727,6 +737,7 @@ if (enableUseMemoCacheHook) {
   createFunctionComponentUpdateQueue = () => {
     return {
       lastEffect: null,
+      events: null,
```

Comment:
useMemoCache defines an optional property on the queue hence the compile time check for the flag. But it's not easy to add another compile time check for enableUseEventHook in a way that lets the various permutations of the flags coexist. So I'm opting to just initialize the property on this object in every case for hidden classes and with the assumption that we'll eventually release useEvent (after deciding what to do with naming etc)